superset/connectors/base/models.py

```diff
@@ -60,8 +60,8 @@ class BaseDatasource(AuditMixinNullable, ImportMixin):
     perm = Column(String(1000))
 
     sql = None
-    owners = None
-    update_from_object_fields = None
+    owners = []  # noqa
+    update_from_object_fields = []  # noqa
```

Comment:
I would perhaps leave these as they are and let someone add the correct types later. My motto (that I often violate :laugh ): When adding typing, don't change logic, especially if it requires `# noqa`.